packages/lib-classifier/src/plugins/tasks/DrawingTask/components/tools/DrawingToolRoot.js

```diff
@@ -1,49 +1,57 @@
 import PropTypes from 'prop-types'
-import React, { useState } from 'react'
+import React, { forwardRef, useState } from 'react'
+import draggable from '../components/draggable'
 
-const STROKE_WIDTH = 1.5
-const SELECTED_STROKE_WIDTH = 2.5
+const STROKE_WIDTH = 2
+const SELECTED_STROKE_WIDTH = 3
 
-function DrawingToolRoot ({ children, tool }) {
-  const [ active, setActive ] = useState(true)
+const DrawingToolRoot = forwardRef(({ children, isActive, mark, svg, tool, onDelete }, ref) => {
+  const [ active, setActive ] = useState(isActive)
   const mainStyle = {
     color: tool && tool.color ? tool.color : 'green',
     fill: 'transparent',
     stroke: tool && tool.color ? tool.color : 'green',
-    strokeWidth: active ? SELECTED_STROKE_WIDTH : STROKE_WIDTH
   }
 
-  function onPointerDown (event) {
-    console.log(`${tool.type} clicked`)
-    event.stopPropagation()
-    event.preventDefault()
-    setActive(true)
+  function onKeyDown (event) {
+    switch (event.key) {
+      case 'Backspace': {
```

Comment:
Just as a follow up to the conversation on Slack from last week. I can delete a mark using either space, enter, or backspace _only_ when the delete button is focused. I think I'm fine with this. My concern was the backspace behavior generally when on the site while on the classifier page using Firefox (and other browsers with this behavior). @beckyrother is this ok with you?

Reply:
It should really respond to Backspace when the mark itself has focus, but I've just realised I haven't copied that over from PFE either.

Overriding `window.onKeyDown` is common in key-binding libraries but I'd always advise against using it.

NB. you can also activate the button with the spacebar, which is standard behaviour but overrides the spacebar as PageDown.

Reply:
Having said that, we should probably get rid of the default WebKit focus outline. I guess PFE removes it in the CSS somewhere?
![Screenshot of drawing marks showing a big, blue focus outline around the selected mark.](https://user-images.githubusercontent.com/59547/69336364-3ee2e780-0c56-11ea-94e7-4383c7636400.png)

Reply:
Checking in with @beckyrother again on the delete button behavior (see my comment above).